### ISSUE TITLE
Maintainers wanted

### ISSUE THREAD

**Issue by Download** [Download/preact-helmet#34]
@IrvingArmenta 
@rosenbjerg
@squidfunk 
@DamianoMagrini 
@acangiani
@DannyAziz

Hi Guys,

There was a time, not too long ago, that I was completely in love with Preact. And in fact, I still am. But I have not had the opportunity to use Preact in a while now and I don't see it coming in the very near future so I don't really have the motivation / time at the moment to maintain this project in my spare time... which is a shame if this code is actually helping you guys, which is what I as an Open Source developer dream of.

So maybe some of you guys would be willing to pick up the ball so to speak and help me maintain this thing?
Basically I will give you 'da power' to do what you want with this repo. I'll also try to help set it up so you guys can publish to NPM by yourselves. 

What do you say?

**Comment by DamianoMagrini** [comment]
While I could probably give you a hand in maintaining this, I feel like Preact Helmet doesn't serve much of a purpose anymore. Always staying up to date with React Helmet would be quite demanding; additionally, with `preact/compat` adding just a very slight overhead, React Helmet (almost identical in size to Preact Helmet) would seem a better choice.

What do you think? I'd love to hear your opinion on this.

**Edit:** to clarify my point, I think we could deprecate this library and encourage users to use React Helmet + `preact/compat` instead.

**Comment by Download** [comment]
Yeah maybe that would be best. I have to think about it some more.

**Comment by rosenbjerg** [comment]
I think DamianoMagrini makes a good point. React-helmet is only slighter bigger the Preact-helmet, so size is no longer a reason to maintain this package. ([preact-helmet size](https://bundlephobia.com/result?p=preact-helmet@4.0.0-alpha-3) vs [react-helmet size](https://bundlephobia.com/result?p=react-helmet@6.1.0))
And now that preact-compat is included in preact, just using react-helmet shouldn't be any problem :)
